soul-admin/src/main/java/org/dromara/soul/admin/service/impl/SelectorServiceImpl.java

```diff
@@ -264,4 +271,15 @@ private SelectorData buildSelectorData(final SelectorDO selectorDO) {
         return SelectorDO.transFrom(selectorDO, pluginDO.getName(), conditionDataList);
     }
 
+    private void updateDivideUpstream(final SelectorDO selectorDO) {
```

Comment:
what is mean？

Reply:
The divide selector added through the web console is updated to UpstreamCheckService#UPSTREAM_MAP.